frontend/src/metabase/containers/AddToDashSelectDashModal/AddToDashSelectDashModal.tsx

```diff
@@ -92,7 +92,9 @@ const AddToDashSelectDashModal = ({
   if (shouldCreateDashboard) {
     return (
       <CreateDashboardModal
-        showOnlyPersonalCollections={isQuestionInPersonalCollection}
+        filterPersonalCollections={
+          isQuestionInPersonalCollection ? "only" : undefined
```

Comment:
`undefine` means showing all collections

Reply:
This is intended to match the query we send to the search endpoints `filter_items_in_personal_collection: "only" | "exclude"`

Although its nuance is a little different because in the search endpoints we're filtering the results, which in this case are only dashboards, questions, or datasets (models)

`filterPersonalCollections: "only" | "exclude" | undefined` are meant not only for filtering items, but also collections that are shown in any picker UI too. 

#### collections
![Screenshot 2023-11-10 at 2 23 31 PM](https://github.com/metabase/metabase/assets/1937582/7948aad5-638b-498d-91e8-ea4283e94d1d)

#### search results
![Screenshot 2023-11-10 at 2 24 51 PM](https://github.com/metabase/metabase/assets/1937582/d55576dc-ddf2-43bb-ab9a-29f1e23d4018)

Do you have any suggestion on improving this flag name?